python/setup.py

```diff
@@ -171,38 +171,25 @@ def get_package_data_files():
     return ["relay/std/prelude.rly", "relay/std/core.rly"]
 
 
+# Temporarily add this directory to the path so we can import the requirements generator
+# tool.
+sys.path.insert(0, os.path.dirname(__file__))
+import gen_requirements
+sys.path.pop(0)
+
+requirements = gen_requirements.join_requirements()
```

Comment:
Do you want to print out some details on what the requirements identified are, to help with debugging?

Reply:
what do you mean exactly?

Reply:
Well, just a suggestion that when this is being invoked, and the requirements being generated automatically, it might be useful to have them dumped to stdout during the invocation (perhaps optionally) so someone debugging issues with Python deps can readily see what they were. Not sure if they are exposed in an obvious way elsewhere.

Reply:
I guess, I don't typically expect to see the requirements printed when I run `python setup.py`. are there cases where you do? I could add a comment saying: to produce requirements.txt files, run `python gen_requirements.py` above this line, which is kind of the typical case i'd expect someone to be looking for them.